server/bleep/src/indexes/file.rs

```diff
@@ -96,9 +98,11 @@ impl File {
         let repo_disk_path = builder.add_text_field("repo_disk_path", STRING);
         let repo_ref = builder.add_text_field("repo_ref", STRING | STORED);
         let repo_name = builder.add_text_field("repo_name", trigram.clone());
+        let raw_repo_name = builder.add_bytes_field("raw_repo_name", FAST);
         let relative_path = builder.add_text_field("relative_path", trigram.clone());
 
         let content = builder.add_text_field("content", trigram.clone());
+        let raw_content = builder.add_bytes_field("raw_content", FAST);
```

Comment:
This has the effect of putting *all* file data in a fast field, essentially defeating the point of a fast field. Is it possible to do this without marking `raw_content` as `FAST`, or does the collector strategy require this?

Reply:
Collectors can only work with fast fields. It's not inherently wrong, I think if we can't measure perf regression here, let's roll with this. However, might be a good idea to try and avoid having both `content` and `raw_content`.

Reply:
yes, unfortunately collectors can only access fast fields/facet fields (which might be worth exploring for our `lang` and `repo_name` fields).

another option i have considered is reconstructing the relevant portions of the document from the term indices (+ term dictionary). with our terms here being trigrams, i am not sure if this makes much sense, but we may be able to retain the speed guarantees and avoid the extra field.